mlflow/projects/databricks.py

```diff
@@ -56,7 +56,7 @@ def _get_databricks_run_cmd(dbfs_fuse_tar_uri, run_id, entry_point, parameters):
     container_tar_path = os.path.abspath(os.path.join(DB_TARFILE_BASE,
                                                       os.path.basename(dbfs_fuse_tar_uri)))
     project_dir = os.path.join(DB_PROJECTS_BASE, tar_hash)
-    mlflow_run_arr = list(map(shlex_quote, ["mlflow", "run", project_dir, "--new-dir",
```

Comment:
Missed this originally, we don't have a test that verifies that this command (that we run on Databricks) is a valid MLflow run command. Would be good to add one, I could do it in a follow-up